src/applications/representative-appoint/components/SelectAccreditedRepresentative.jsx

```diff
@@ -111,7 +111,7 @@ const SelectAccreditedRepresentative = props => {
   };
 
   const handleSelectRepresentative = async selectedRepResult => {
-    if (selectedRepResult === currentSelectedRep) {
+    if (selectedRepResult === currentSelectedRep && isReviewPage) {
```

Comment:
```js
  const currentSelectedRep = useRef(
    formData?.['view:representativeSearchResults'],
  );
```

Would `currentSelectedRep` always be an array of search results? If so, this conditional could never be true.

Reply:
Should be fixed.